### ISSUE TITLE
Renaming text needs to be changed. 

### ISSUE THREAD

**Issue by pxwee5** [flarum/core#428]
When renaming titles, the notification is hard to read. 

>Dominion changed the title from A couple things - Pinning Sidebar, Post Numbers etc to A few things - Pinning Sidebar, Post Numbers etc..

This is not exactly the easiest to read. Perhaps bolding the titles and properly paragraphing them ...

>Dominion changed the title 
>from **"A couple things - Pinning Sidebar, Post Numbers etc"**
>to **"A few things - Pinning Sidebar, Post Numbers etc.."**

Low priority stuff but it is definitely easier on the eyes.

**Comment by franzliedke** [comment]
Again asking for your input, @dcsjapan.
As far as I know, we cannot have the HTML line breaks in the translation files (and I'd argue we shouldn't). Can we safely separate this into three strings, assuming it won't be a problem for other languages? Typical [stumbling block 2 situation](http://discuss.flarum.org/d/336-common-l10n-stumbling-blocks/5), I'd say...

**Comment by dcsjapan** [comment]
@franzliedke Glad to help!

I think three strings should be fine here. If the "from ... to ..." syntax doesn't work in a language, then a clever translator should be able to come up with an alternative, such as:

- Dominion changed the title.
- old: "A couple things - Pinning Sidebar, Post Numbers etc"
- new: "A few things - Pinning Sidebar, Post Numbers etc.."

... or something like that.

However, it would be a *very* good idea to make the second and third lines phrases, since in some cases the "from" and or the "to" will have to come *after* the thread title, not before them.

**Comment by franzliedke** [comment]
How about only showing the old and new title when the user clicks on a little icon / text?
I agree that the long form is mostly confusing and not very glanceable. The three-line form is kinda long, though. Making it toggle-able (ugh) would solve that problem.

**Comment by tobscure** [comment]
True, good idea

**Comment by tobscure** [comment]
@franzliedke I think we should change these posts to appear as follows:

> **Toby** changed the title to **This is the new title**.

* We should definitely show the new title without requiring interaction, as it's pretty important information for context. It sets the "topic" for the viewer as they read the posts which follow. Just having it read "Toby changed the title" is like reading "Toby changed the tags" – not very useful!
* The old title is not so important, but should still be accessible. To save a click, we can show it in a tooltip when hovering over the post: **Changed from "This is the old title"**
* Bold for the title instead of italics for a cleaner look, and to better distinguish the title from the surrounding text.

Sorry that I'm only thinking about this properly now! It's the kind of thing that I just needed to see to realise that it's not quite right :)

**Comment by dcsjapan** [comment]
I agree, putting the new title up front makes sense.

The tooltip wording seems a bit awkward to me. How about something like:

> The old title was: "This is the old title".

... or drop the quote marks and use boldface, if that's possible in a tooltip.

**Comment by franzliedke** [comment]
Okay, I did it as you suggested now.

:+1: Much simpler.
:-1: I spent so much time on the old implementation that it was a pain to throw away. ;)

There seems to be some kind of bug with escaping the double quotes, though. AFAIK [this commit](https://github.com/flarum/flarum-ext-english/commit/2d3a397b97267bcb5c4fedbd32df683536b09bdd) should work just fine, however in my browser I see extra commas being inserted after the leading and before the trailing double quote. Could that be because of our translator implementation messing something up?
